src/main/controllers/CrimeLoginController.ts

```diff
@@ -0,0 +1,15 @@
+import { PipRequest } from '../models/request/PipRequest';
+import { Response } from 'express';
+import { CRIME_IDAM_URL, FRONTEND_URL } from '../helpers/envUrls';
+
+export default class CrimeLoginController {
+    public async get(req: PipRequest, res: Response): Promise<void> {
```

Comment:
I think the params declaration below can be moved outside of the get method, as it doesn't use anything from the req/res which would save it being called multiple times.

Reply:
I tried to move above but it is not working. Somehow it is not able to ready the env variable and unit test was failing. So I reverted the changes.